app/src/main/java/org/mozilla/fenix/settings/search/SearchEngineListPreference.kt

```diff
@@ -234,7 +231,7 @@ abstract class SearchEngineListPreference @JvmOverloads constructor(
                 it.identifier != engine.identifier
             },
             default = if (searchEngineList.default?.identifier == engine.identifier) {
-                null
```

Comment:
This was why the UI test was failing 🙃 changing it back to null so that the default is set correctly. My change was incorrectly overriding when someone chose a non-custom default